src/main/views/case-audit/template.njk

```diff
@@ -125,6 +127,7 @@
     </div>
 
   </div>
+  {{ csrfProtection(csrfToken) }}
```

Comment:
Is there a reason you placed it here, in the middle of the form? Personally, I would prefer right after <form> opening, so that it would be immediately clear that there is csrf token added to the form.

Reply:
for me its a habit of placing it above the submission button. i can move it to the top if you prefer.